modules/articles/client/views/form-article.client.view.html

```diff
@@ -13,8 +13,8 @@
           </div>
         </div>
         <div class="form-group">
-          <label class="control-label" for="content">Content</label>
-          <textarea name="content" data-ng-model="vm.article.content" id="content" class="form-control" cols="30" rows="10" placeholder="Content"></textarea>
+          <label for="content">Content</label>
+          <summernote name="content" config="summeroptions" ng-model="vm.article.content" id="content"></summernote>
```

Comment:
We don't want to make our UI bloated and opinionated when most people aren't using MEAN for its UI. As I've said in other PRs, our focus at this time is not UI, so we don't want to put effort into that part of the framework right now.